### PR TITLE
chore(android/app): Update whatsnew with available display languages

### PR DESCRIPTION
**Pull request by darcywong00**
* List available display languages
* ~Add KMEA breaking changes (Most of the items from https://github.com/keymanapp/keyman/wiki/Keyman-13.0-to-14.0-Migration#android)~

### CODE REVIEW

File: android/help/about/whatsnew.md [hunk 1]
```diff
@@ -12,9 +12,43 @@ Here are some of the new features we have added to Keyman for Android 14.0:
...
+dependencies {
+    implementation 'io.sentry:sentry-android:3.1.0'
+    implementation 'androidx.preference:preference:1.1.1'
+}
+```
+* Deprecated KMManager APIs
+    * `KMManager.showLanguageList()` - KMEA no longer uses cloud keyboard catalog, so showLanguageList() no longer displays. (Embedded browser used to search for keyboards)
+    * `KMManager.KMKey_CustomKeyboard` - "Custom" property for keyboards no longer tracked
+    * `KMManager.KMKey_CustomModel` - "Custom" property for lexical models no longer tracked
+    * `addKeyboard` - Deprecate syntax using `HashMap<string, string>`. Replace with `<Keyboard>`
```
Comment (mcdurdin) [inline android/help/about/whatsnew.md:54]: Can we put Engine stuff into the Developer Engine reference instead?

File: android/help/about/whatsnew.md [hunk 2]
```diff
@@ -12,9 +12,43 @@ Here are some of the new features we have added to Keyman for Android 14.0:
 * Added new menu to add languages for an installed keyboard package (#3255)
 * Consolidated install menus for installing keyboards (#3245)
 * Fix slow input in the embedded browser (#3768)
-* Add system globe action to show system keyboards (#3197)
+* Add system globe action `GLOBE_KEY_ACTION_SHOW_SYSTEM_KEYBOARDS ` to show system keyboards (#3197)
```
Comment (mcdurdin) [inline android/help/about/whatsnew.md:15]: This belongs in Engine reference, not helpful for end users :)

**Review by mcdurdin — CHANGES_REQUESTED** [review]

File: android/help/about/whatsnew.md (hunk 1 above)
Comment (darcywong00) [reply android/help/about/whatsnew.md:54]: So should there be a new page on http://help.keyman-staging.com/developer/engine/android/14.0/ for "whatsnew"?
Most of the above content is geared for existing keyboard app developers that would need to make changes going from KMEA 13 to 14

Comment (mcdurdin) [reply android/help/about/whatsnew.md:54]: That sounds good to me. The user documentation should not contain details relevant only to developers :grin:

**Review by mcdurdin — APPROVED** [review]

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 14.0.253-beta](https://keyman.com/downloads/releases/beta/14.0.253)

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.19-alpha](https://keyman.com/downloads/releases/alpha/15.0.19)

**Pull request merged**